database/migrations/2021_12_11_110354_populate_clients.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+class PopulateClients extends Migration
+{
+/**
+* Run the migrations.
+*
+* @return void
+*/
+public function up()
+{
+DB::statement("
+INSERT INTO clients (first_name, last_name) 
```

Comment:
Please avoid inserts for clients and about_clients

Reply:
fixed